### PR TITLE
Remove old floodlight implementation

### PR DESCRIPTION
**Pull request by remybach**
🐿 v2.11.0

### CODE REVIEW

**Comment by remybach** [comment]
Oops... you made me realise this would've disabled all tracking 🤦‍♂️ Will fix.

**Comment by remybach** [comment]
@liamkeaton To your point though, I looked over the ticket again and Bastien did say just before he left that it should be getting sent through so I've added it in to be 100% sure it's going up.

File: components/n-ui/tracking/third-party/floodlight.js [hunk 1]
```diff
@@ -2,19 +2,15 @@
 module.exports = function (flags) {
 
 	const isAnonymous = !/FTSession/.test(document.cookie);
-	const spoorId = /spoor-id=([^;]+)/.exec(document.cookie);
-	const host = 'https://4235225.fls.doubleclick.net/activityi;src=4235225';
+	const spoor = /spoor-id=([^;]+)/.exec(document.cookie);
+	const spoorId = (spoor && spoor.length > 0) ? spoor[1] : '';
 
 	// sign-up attributes
 	const c = document.querySelector('[data-signup-country]');
```
Comment (liamkeaton) [inline components/n-ui/tracking/third-party/floodlight.js]: Where are these being used?

Comment (remybach) [reply components/n-ui/tracking/third-party/floodlight.js]: Oops... how did those get back in there. Good spot 👍

**Pull request merged**